### PR TITLE
Added support to change timezone from settings page (#1x69rpc)

### PR DESCRIPTION
**Pull request by Yashi002**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -32,12 +31,12 @@
 </template>
 
 <script lang="ts">
-import { IonApp, IonContent, IonHeader, IonItem, IonLabel, IonList, IonMenu, IonNote, IonRadio, IonRouterOutlet, IonTitle, IonToolbar } from '@ionic/vue';
+import { IonApp, IonContent, IonHeader, IonItem, IonLabel, IonList, IonMenu, IonNote, IonRadio, IonRouterOutlet, IonTitle, IonToolbar, alertController } from '@ionic/vue';
 import { defineComponent } from 'vue';
 import { loadingController } from '@ionic/vue';
+import { useStore } from './store'
```
Comment (ymaheshwari1) [inline src/App.vue:37]:
```suggestion
import { useStore } from '@/store'
```

File: src/views/timezone-modal.vue [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  data() {
+    return {
+      queryString: '',
+      filteredTimeZones: [],
+      timeZones: [],
+      timeZoneId: ''
+    }
+  },
+  computed: {
+  },
```
Comment (ymaheshwari1) [inline src/views/timezone-modal.vue:77]: Removed computed option as it's not used.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]
Improve indentation

File: src/App.vue (hunk 1 above)
Comment (Yashi002) [reply src/App.vue]: Done.

File: src/views/timezone-modal.vue (hunk 2 above)
Comment (Yashi002) [reply src/views/timezone-modal.vue]: Removed unused computed

**Comment by Yashi002** [comment]
Closing this PR, as different branch got merged in origin while checking and unwanted changes are coming due to that

**Pull request closed without merging**